File: Data Structures & Algorithms/two-integer-sum/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        let m = new Map()
        for (let i = 0; i < nums.length; i++){
            const num = nums[i]
            if (m.has(target - num)){
               return [m.get(target - num), i]
            }
            m.set(num, i)
        }
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        res = [0] * (n * 2)
        for i, num in enumerate(nums):
            res[i] = num
            res[i + n] = num

        return res